drivers/input/i8042prt/pnp.c

```diff
@@ -263,6 +263,50 @@ i8042DetectMouse(
     INFO_(I8042PRT, "Mouse not detected\n");
 }
 
+static void
+i8042DetectMouseType(
+    IN PPORT_DEVICE_EXTENSION DeviceExtension)
+{
+    int chkcnt = 10;
```

Comment:
If you're going re-write my loop, you need to delete the variable from it, as well

Reply:
okay